doc/software-design-development-guidance/index.md

```diff
@@ -0,0 +1,267 @@
+# Software design development guidance
+
+!!! note
+    Work in Progress
+
+**Status**: first sketch, work in progress, request for collaboration
+
+**Date**: 2025-03-31
+
+**Governance**: To Be Discovered; potentially a combo of this repo partipants, DHCW CISO, NHS Wales UCB peers, etc.
+
+## Situation - Context and Problem Statement
+
+Broadly we need to design and develop many software products for patients and healthcare workers, and we want our design and development to emphasize usability, accessibility, maintainability, and similar system quality attributes.
+
+Narrowly we need to immediately design and develop the emergency department module, and we need to give the program some UI/UX guardrails:
+
+* Design system: e.g. how should a top navigation menu look and feel to the user.
+  
+* Development system: e.g. how should a programmer implement the menu with JavaScript/TypeScript.
+ 
+* Testing system: e.g. how to test the menu for accessibility, multi-language capability, etc.
+
+## Background - Decision Drivers
+
+We want to get good-enough quality work in place now, so we can try it out, ask for feedback, and seek help from peers and advisors.
+
+We want to align with existing design-development recommends, not reinvent the wheel.
+
+We generally favor considering systems that we are confident are relevant and ready:
+
+* Government guidelines: GOV.UK Design System, NHS.UK Design System, 18F Design System, others?
+
+* Business specifications: Apple Human Interfaces Guidelines, Google Material Design, Microsoft Fluent Design System, others?
+
+* Free open source components using JavaScript and TypeScript: shadcn, Flowbite, SkeletonUI, others?
+
+## Assessment - Considered Options
+
+We generally favor considering systems that we are confident are relevant and ready:
+
+[Introductory summaries are by Wix](https://www.wix.com/studio/blog/design-system-examples)
+
+Government guideline options:
+
+* [GOV.UK Design System](https://design-system.service.gov.uk/)
+
```

Comment:
Can we add https://service-manual.ons.gov.uk/design-system - some of us were involved in the inception of this.

Reply:
Yes done